graphql/src/schema/api.rs

```diff
@@ -174,6 +216,26 @@ fn add_block_height_type(schema: &mut Document) {
     schema.definitions.push(def);
 }
 
+fn add_subgraph_error_policy_type(schema: &mut Document) {
```

Comment:
Not sure if you noticed, but I added a `meta.graphql` file a while ago. Would be cleaner to add this type definition to that file, too, since a GraphQL schema definition is much easier to read than Rust code that builds the AST.

Reply:
That is nice! Moved it there.